src/dpp/discordvoiceclient.cpp

```diff
@@ -1166,20 +1166,35 @@ discord_voice_client& discord_voice_client::set_send_audio_type(send_audio_type_
 
 discord_voice_client& discord_voice_client::send_audio_raw(uint16_t* audio_data, const size_t length)  {
 #if HAVE_VOICE
-	const size_t max_frame_bytes = 11520;
+	static const size_t max_frame_bytes = SEND_AUDIO_RAW_MAX_LENGTH;
+
+	if (length < 4) {
+		return *this;
+	}
+
+	if ((length % 4) != 0) {
+		throw dpp::voice_exception("Raw audio packet size should be left with zero remainder when divided by 4");
```

Comment:
This is confusing. I suggest "should be divisible by 4" or "should be a multiple of 4"

Reply:
I'm also confused, you have this exception here but if we send a size of 3 it just silently doesn't do anything? Shouldn't it send an error too if length < 4?